meson_options.txt

```diff
@@ -33,6 +33,11 @@ option('examples',
     value: false,
     description: 'Enable building examples')
 
+option('png',
```

Comment:
I think the png option should be elements of the loaders' options.

Reply:
@MathewP1  ping?